packages/core/src/modules/proofs/ProofsApi.ts

```diff
@@ -495,7 +495,7 @@ export class ProofsApi<PPs extends ProofProtocol[]> implements ProofsApi<PPs> {
           serviceParams: {
             service: recipientService.resolvedDidCommService,
             senderKey: ourService.resolvedDidCommService.recipientKeys[0],
-            returnRoute: true,
+            returnRoute: options.hasReturnRoute !== false, // defaults to true if missing
```

Comment:
Thinking of it more. What if we just disable return routing for the acceptPresentation call? It's the end of the protocol and we never expect a return. For the accept request it makes sense as we expect a presentation back. This way we don't have to add any new parameters but we set returnRoute to:
- `true` in acceptRequest (ProofsApi)
- `false` in `acceptPresentation (ProofsApi)
- `true` in acceptOffer (CredentialsApi)
- `true` in acceptRequest (CredentialsApi)
- `false` in acceptCredential (CredentialsApi)

What do you think?

Reply:
The `acceptOffer `and `acceptRequest `in credentials are already set to true so we are ok there. I have set `acceptPresentation `and `acceptCredential `to false as it is quite true, since it's the end of the protocol no need for it to be anything else.

I would like to leave `acceptRequest `in ProofsAPI with the option to pass in a value as I think for AFJ <-> AFJ we might have an ack and could need this set to true, but for aca-py <-> AFJ we would need it set to false as no ack (at least according to my colleagues who have been doing the testing).

There was a suggestion from @JamesKEbert to set the `returnRoute `flag according to the `pleaseAck `decorator value but I thought this was overkill so I didn't do it that way.